requirements.txt

```
@@ -1,6 +1,6 @@
 api-client==1.3.1
 asgiref==3.5.2
-certifi==2022.12.7
+certifi==2023.07.22
```

Might be better to make it permissive.

```suggestion
certifi>=2023.07.22
```